s18/use_Gadget.rb

```diff
@@ -0,0 +1,16 @@
+require_relative "classes/gadget.rb"
```

Comment:
What's the difference between `require` and `require_relative`?

Reply:
Both serve to call another file, but require has to cite the absolute path while requirer require_relative only the path from where the file is located from where it's called.